src/web/app/components/question-types/question-statistics/mcq-question-statistics.component.html

```diff
@@ -8,7 +8,7 @@
   </div>
   <div class="row">
     <div class="col-sm-12 table-responsive">
-      <tm-sortable-table [columns]="summaryColumnsData" [rows]="summaryRowsData" [initialSortBy]="SortBy.MCQ_PERCENTAGE"></tm-sortable-table>
+      <tm-sortable-table [columns]="summaryColumnsData" [rows]="summaryRowsData"></tm-sortable-table>
```

Comment:
Since we need to sort by the choice/option, should pass in the initial SortBy to be `SortBy.MCQ_CHOICE`

Reply:
Because the original order of the options may not be alphabetical. If we sort by MCQ choice the default ordering can be lost. This may not be intended in some cases I think.